docs/hub/repositories.md

```diff
@@ -0,0 +1,210 @@
+- Transferring a repository from user A to user B.
+
+If these are use cases you need help with, please send us an email at **website at huggingface.co**.
+
+
+### Dealing with large files
+
+Do you have files larger than 10MB? Those files should be tracked with `git-lfs`. When you use Hugging Face to create a repository, we automatically provide a list of common file extensions for these sfiles in `.gitattributes`, which `git-lfs` uses to efficiently track changes to your large files. However, you might need to add new extensions if your file types are not already handled. You can do so with `git lfs track "*.your_extension"`.
+
+
+## Best practices
+
+### Private Repositories
+
+You can choose a repository's visibility when you create it, and any repository that you own can have its visibility toggled between *public* and *private* in the **Settings** tab. Unless your repository is owned by an organization (more about that [**here!**](TODO)), you are the only user that can make changes to your repo or upload any code. Setting your visibility to *private* will ensure that your repo is not discoverable for other users by searching on the Hub, and that anyone other than you who visits the URL for your repo will receive `404 - Repo not found` error. Additionally, other users will not be able to clone your repo.
+
+### Security for repositories
+
+`git` has an authentication layer to control who can push commits to a repo, but it does not authentify the actual commit authors.
+
+In other words, you can commit changes as `Elon Musk <elon@tesla.com>`, push them to your preferred `git` host (for instance github.com) and your commit will link to Elon's GitHub profile. (Try it! But don't blame us if Elon gets mad at you for impersonating him)
+
+See this post by Ale Segala for more context: [How (and why) to sign `git` commits](https://withblue.ink/2020/05/17/how-and-why-to-sign-git-commits.html)
+
+You can prove a commit was authored by you, using GNU Privacy Guard (GPG) and a key server. GPG is a cryptographic tool used to verify the authenticity of a message's origin. To learn about setting up signed commits verification with GPG, visit the [**Security**](TODO) section of our docs!
+
+### Licenses
+
+You are able to add a license to any repo that you create on the Hugging Face Hub to let other users know about the permissions that you want to attribute to your code. Likewise, remember to seek out and respect a project's license if you're considering using their code.
+
+A [**full list of the available licenses**](TODO) is available in these docs.
+
+## Next steps
+
+A good place to visit if you want to continue learning about Git is [GitHub's "Using Git" tutorial](https://docs.github.com/en/get-started/using-git/about-git). These next sections highlight features and additional information that you may find useful to make the most out of the Git repositories on the Hugging Face Hub.
```

Comment:
I think https://learngitbranching.js.org/ is more fun :smile:

Reply:
I saw it was linked below, I think Git Branching is not just for branches, but for everything, so I think it's a good resource here as well